models/cice/shell_scripts/CESM2_1_DART_config

```diff
@@ -0,0 +1,413 @@
+
+if ( ! -e ./xmlquery ) then
+   echo "ERROR: $0 must be run from a CASEROOT directory".
+   exit -1
+endif
+
+setenv CASE          `./xmlquery --value CASE     `
+setenv CASEROOT      `./xmlquery --value CASEROOT `
+setenv COMPSET       `./xmlquery --value COMPSET  `
+setenv EXEROOT       `./xmlquery --value EXEROOT  `
+setenv RUNDIR        `./xmlquery --value RUNDIR   `
+setenv NINST_ICE     `./xmlquery --value NINST_ICE`
+setenv ICE_COMPONENT `./xmlquery --value COMP_ICE `
+
+echo $CASE
+echo $CASEROOT
+# Check to make sure we are running what we are supporting
+
+if ( $ICE_COMPONENT != 'cice' ) then
+   echo 'ERROR: This configuration file is specifically for "cice".'
+   echo 'ERROR: the ice component for this case is "'${ICE_COMPONENT}'".'
+   exit 1
+endif
+
+set num_instances = $NINST_ICE
+
+./xmlchange DATA_ASSIMILATION_ICE=TRUE
+./xmlchange DATA_ASSIMILATION_CYCLES=1
+./xmlchange DATA_ASSIMILATION_SCRIPT=${CASEROOT}/assimilate.csh
+
+# /glade/u/home/yfzhang/dart_rma_trunk and /glade/p/work/yfzhang/observations/syn/cice5/member10/aggre/aice are replaced by useful
+# values by the CESM_configure script.
+# It should reference the
+# base portion of the DART code tree. 
+
+setenv DARTROOT  BOGUS_DART_ROOT_STRING
+setenv OBSROOT   BOGUS_DART_OBS_STRING
+
+# ==============================================================================
+# Some
+# ==============================================================================
+
+set nonomatch       # suppress "rm" warnings if wildcard does not match anything
+
+# The FORCE options are not optional.
+# The VERBOSE options are useful for debugging though
+# some systems don't like the -v option to any of the following
+switch ("`hostname`")
+   case ys*:
+         # NCAR "yellowstone"
+         set   MOVE = '/bin/mv -v'
+         set   COPY = '/bin/cp -v --preserve=timestamps'
+         set   LINK = '/bin/ln -vs'
+         set REMOVE = '/bin/rm -rf'
+      breaksw
+   case be*:
+         # NCAR "bluefire"
+         set   MOVE = '/usr/local/bin/mv -v'
+         set   COPY = '/usr/local/bin/cp -v --preserve=timestamps'
+         set   LINK = '/usr/local/bin/ln -vs'
+         set REMOVE = '/usr/local/bin/rm -rf'
+      breaksw
+   case r*:
+          set   MOVE = '/usr/local/bin/mv -v'
+          set   COPY = '/usr/local/bin/cp -v --preserve=timestamps'
+          set   LINK = '/usr/local/bin/ln -vs'
+          set REMOVE = '/usr/local/bin/rm -rf'
+   default:
+         # NERSC "hopper"
+         set   MOVE = 'mv -v'
+         set   COPY = 'cp -v --preserve=timestamps'
+         set   LINK = 'ln -vs'
+         set REMOVE = 'rm -rf'
+      breaksw
+endsw
+
+echo ""
+
+# ==============================================================================
+# make sure the required directories exist
+# VAR is the shell variable name, DIR is the value
+# ==============================================================================
+
+foreach VAR ( CASEROOT DARTROOT )
+   set DIR = `eval echo \${$VAR}`
```

Comment:
looks like eval is being used to escape the variable here, there are better ways to do this.